engines/wintermute/base/gfx/osystem/base_surface_osystem.cpp

```diff
@@ -433,7 +433,8 @@ bool BaseSurfaceOSystem::drawSprite(int x, int y, Rect32 *rect, Rect32 *newRect,
 	// But no checking is in place for that yet.
 
 	// Optimize by not doing alpha-blits if we lack alpha
-	if (_alphaType == Graphics::ALPHA_OPAQUE && !transform._alphaDisable) {
```

Comment:
Why has the check for transform._alphaDisable been removed?

Reply:
It was a `if(b!=true){b=true;}` pattern, which has equal effect to just `{b=true;}`, but was harder to read in line of several checks and I was not sure if it is performance-critical here to use `if(b!=true){b=true;}` for a single boolean.